src/panels/media-browser/ha-bar-media-player.ts

```diff
@@ -212,7 +212,7 @@ export class BarMediaPlayer extends SubscribeMixin(LitElement) {
     const mediaDescription = computeMediaDescription(stateObj);
     const mediaDuration = formatMediaTime(stateObj.attributes.media_duration);
     const mediaTitleClean = cleanupMediaTitle(
-      stateObj.attributes.media_title || ""
+      stateObj.attributes.media_title || stateObj.attributes.media_content_id
```

Comment:
Moved up here so that `mediaTitleClean` already has the final value as further down this var is a condition to show the secondary info such as artist.

Reply:
Why do we use `mediaTitleClean` as a condition to show `mediaDescription`? 🤔 (not related to your changes)